### ISSUE TITLE
Do .ips files work?

### ISSUE THREAD

**Issue by kevosmin** [Wh0ba/XPatcher#11]
I’m unable to import an ips file because it’s non-selectable from the files app

**Comment by lonkelle** [comment]
@kevosmin Yeah, same here. The workaround is to "share" the file to XPatcher from the Files app instead of importing it from within XPatcher.

**Comment by lonkelle** [comment]
@kevosmin Yeah, same here. The workaround is to "share" the file to XPatcher from the Files app instead of importing it from within XPatcher.

**Comment by lonkelle** [comment]
@kevosmin Yeah, same here. The workaround is to "share" the file to XPatcher from the Files app instead of importing it from within XPatcher.
